jdaviz/components/viewer_data_select.vue

```diff
@@ -273,6 +273,10 @@ module.exports = {
     isRefData() {
       return this.$props.item.viewer.reference_data_label === this.$props.item.name
     },
+    isChild(item) {
+      // only override multi_select choice when data entry is a child, and is in cubeviz:
+      return (this.$props.viewer.config === 'cubeviz' && item.is_child)
```

Comment:
this might actually make sense in Imviz as well (when overriding to single select, toggling the DQ layers independently of that choice might make sense?)

Reply:
still think this is worth considering (and if not, maybe the method name should be changed to mention cubeviz as well?)